features/FEATURE_LWIP/lwip-interface/lwip/src/netif/ppp/lwip_ppp.c

```diff
@@ -583,6 +583,10 @@ static err_t ppp_netif_output(struct netif *netif, struct pbuf *pb, u16_t protoc
   }
 #endif /* CCP_SUPPORT */
 
+#if PRINTPKT_SUPPORT
```

Comment:
Not sure we should actually be committing these core debugging changes to lwIP - they were useful, but I'd kind of prefer to not be fiddling with the lwIP core.